shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -38,7 +39,7 @@
 }
 
 UIView<UITextInput>* AccessibilityBridge::textInputView() {
-  return [platform_view_->GetTextInputPlugin() textInputView];
+  return [[platform_view_->GetOwnerViewController().get().engine textInputPlugin] textInputView];
```

Comment:
I don't recommend this change because of the law of dimeter.  I recommend placing this code inside the GetTextInputPlugin() method.

Reply:
I generally agree if our current components lifecycles and ownerships are cleanly designed and encapsulated. In practice, I think the platform view class shouldn't exist and I'd like to gradually hollow it out more and more. We're also already not minimizing knowledge in this class a few lines above with the binaryMessenger.